src/test/java/com/github/dockerjava/cmd/LogContainerCmdIT.java

```diff
@@ -197,4 +198,54 @@ public void asyncLogContainerWithSince() throws Exception {
 
         assertThat(loggingCallback.toString(), containsString(snippet));
     }
+
+    @Test
+    public void asyncLogContainerWithDetails() throws Exception {
+        CreateContainerResponse container = dockerRule.getClient().createContainerCmd("busybox")
+                .withCmd("--log-driver", "journald", "--log-opt", "env=ENV_VAR", "-e", "ENV_VAR=logtester.1234", "flyinprogrammer/logtester")
```

Comment:
how does this work?

Reply:
If run with docker, the command should be `docker run --log-driver journald --log-opt env=SOME_ENV_VAR -e SOME_ENV_VAR=logtester.1234 flyinprogrammer/logtester`. I'm not sure how to set it with `.withCmd`.

Reply:
CMD is a CMD for binary
What you did is `docker run busybox sh --log-driver journald --log-opt env=SOME_ENV_VAR -e SOME_ENV_VAR=logtester.1234 flyinprogrammer/logtester`
createContainer() is a java command 
createContainer().withDetails() ? Look at examples in existing test case.